spec/requests/access_request_request_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe "Access Requests", type: :request do
+  describe 'POST #approve' do
+    let(:credentials) { ActionController::HttpAuthentication::Basic.encode_credentials("super.admin@education.gov.uk", "beta") }
+    let(:auth_header) { { 'HTTP_AUTHORIZATION' => credentials } }
+    it 'succeeds' do
```

Comment:
I know naming is hard, but can we try something like "posts request to backend" here? 😸

Reply:
Yes, braincells were deployed on making it work at the time and forgot to return to this, good spot.

Reply:
done, thanks for the suggested text.